app/src/ui/dialog/header.tsx

```diff
@@ -63,7 +63,7 @@ export class DialogHeader extends React.Component<IDialogHeaderProps, {}> {
     // I don't know and we may want to revisit it at some point but for
     // now an anchor will have to do.
     return (
-      <a className="close" onClick={this.onCloseButtonClick}>
+      <a className="close" onClick={this.onCloseButtonClick} aria-label="close" role="button">
```

Comment:
This change looks good, but because the line is a bit longer our linting tool is reporting a formatting issue:

```
$ prettier --check "./**/*.{ts,tsx,js,jsx,scss,html,yaml,yml}"
Checking formatting...
app/src/ui/dialog/header.tsx
Code style issues found in the above file(s). Forgot to run Prettier?
```

You can fix this locally by running `yarn lint:fix` and committing the change, which should look something like this:

```diff
diff --git a/app/src/ui/dialog/header.tsx b/app/src/ui/dialog/header.tsx
index a6ffe2d34..456b177bd 100644
--- a/app/src/ui/dialog/header.tsx
+++ b/app/src/ui/dialog/header.tsx
@@ -63,7 +63,12 @@ export class DialogHeader extends React.Component<IDialogHeaderProps, {}> {
     // I don't know and we may want to revisit it at some point but for
     // now an anchor will have to do.
     return (
-      <a className="close" onClick={this.onCloseButtonClick} aria-label="close" role="button">
+      <a
+        className="close"
+        onClick={this.onCloseButtonClick}
+        aria-label="close"
+        role="button"
+      >
         <Octicon symbol={OcticonSymbol.x} />
       </a>
     )
```